### PR TITLE
librime: 1.5.0 -> 1.5.3

### PR DESCRIPTION
**Pull request by xrelkd**
###### Motivation for this change

Update `librime` to the latest release.
https://github.com/rime/librime/releases/tag/1.5.3

**Note**: for reviewer, it is suggested that we can use `@GrahamcOfBorg build librime fcitx-engines.rime brise` to build this library.

Thank you!

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build librime fcitx-engines.rime brise

**Comment by xrelkd** [comment]
@FRidh Thanks for the review, this PR has been updated again.
Please call `ofborg` to build it again, thank you very much!

File: pkgs/development/libraries/librime/default.nix [hunk 1]
```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
   pname = "librime";
-  version = "1.5.0";
+  version = "1.5.2";
```
Comment (marsam) [inline pkgs/development/libraries/librime/default.nix]: 1.5.3 was released recently, would you mind updating it? https://github.com/rime/librime/releases/tag/1.5.3

Comment (xrelkd) [reply pkgs/development/libraries/librime/default.nix]: Thanks for the review.
`librime` has been updated to the latest release.

**Comment by marsam** [comment]
@GrahamcOfBorg build librime fcitx-engines.rime brise

btw can you ask to be included in extra-known-users in https://github.com/NixOS/ofborg? so it allows you to call @GrahamcOfBorg

**Comment by xrelkd** [comment]
@marsam Thank you for the invitation!
It will be helpful if I could call @GrahamcOfBorg in the future.

**Comment by marsam** [comment]
Thanks! tested locally on NixOS

**Pull request merged**